lms/static/js/groups/views/cohort_discussions_inline.js

```diff
@@ -17,10 +17,11 @@
                     },
 
                     render: function () {
-                        var alwaysCohortInlineDiscussions = this.cohortSettings.get('always_cohort_inline_discussions');
+                        var alwaysCohortInlineDiscussions = this.cohortSettings.get('always_cohort_inline_discussions'),
+                            inline_discussions = this.model.get('inline_discussions');
 
                         this.$('.cohort-inline-discussions-nav').html(this.template({
-                            inlineDiscussionTopics: this.getInlineDiscussionsHtml(this.model.get('inline_discussions')),
+                            inlineDiscussionTopicsHtml: this.getInlineDiscussionsHtml(inline_discussions),
```

Comment:
In the template, use `HtmlUtils.ensuerHtml()` as you mentioned.

For, this, I would fix `getInlineDiscussionsHtml()` to also use HtmlUtils calls as appropriate to return an HTML snippet.